modules/hosted-che/partials/proc_creating-a-workspace-from-the-user-dashboard.adoc

```diff
@@ -18,7 +18,7 @@ This section describes how to create a workspace from the user dashboard in Ecli
 
 . Once login, the redirect to the user dashboard will happen.
 
-. Click the btn:[Add Workspace] button and create a "Custom Workspace" using Devfile, or choose the "Get Started" tab, and pick one of the existing samples.
+. Click the btn:[Create Workspace] menu item and create a "Custom Workspace" using Devfile, or use the default "Quick Add" tab, and pick one of the existing samples.
```

Comment:
Please reword as: Click the btn:[Create Workspace] and create a "Custom Workspace" using Devfile, or use the default "Quick Add" tab, and select one of the existing samples.